src/component/Calendar.jsx

```diff
@@ -0,0 +1,98 @@
+import React, { useEffect, useState } from "react";
+import { Calendar, dayjsLocalizer } from "react-big-calendar";
+import dayjs from "dayjs";
+import "react-big-calendar/lib/css/react-big-calendar.css";
+import { apiPath } from "../api";
+
+function MyCalendar() {
+  const localizer = dayjsLocalizer(dayjs);
+  const [events, setEvents] = useState([]);
+
+  const fetchMembers = async (teamId) => {
+    try {
+      const responseMembers = await fetch(apiPath(`/members`));
+      const membersData = await responseMembers.json();
+
+      const teamMembers = membersData.filter(
+        (member) => member.team_id === teamId
+      );
+
+      return teamMembers;
+    } catch (error) {
+      throw new Error("Error fetching members:", error);
+    }
+  };
+
+  useEffect(() => {
+    const fetchEvents = async () => {
+      try {
+        const teamCode = window.location.pathname.split("/").pop();
```

Comment:
I am not sure I understand why you need to get the teamCode like this here. 
It is good problem solving skills and creative, but perhaps it is possible to role back a bit? 
So either I am too tired or it can be done simplier 😴 

Looks like you have the team id in the parent container to the Calendar, so why not pass it as prop?
If I overlooked or misunderstood something, you could then use useParams instead of the pathname approach 🙏🏽